### ISSUE TITLE
chore: make accessible

### ISSUE THREAD

**Issue by henrylin03** [henrylin03/eqa-martial-arts#12]
# potential improvements

- [x] on wider screens, the 'classes' nav link appears on _hover_ - however, this is not very "operable" (WCAG principle) by non-mouse users - https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/
![Image](https://github.com/user-attachments/assets/426af8a8-f61f-4f41-acee-d672cf1a9ee1)
- [x] tabbing available for header links
- [x] tabbing available for footer links
- [x] tabbing outline colour should be aligned with theme (currently, it's the default Mantine UI library blue)
- [ ] #53

# useful resources

- wcag 2 checklist with recommendations: https://webaim.org/standards/wcag/checklist

**Comment by henrylin03** [comment]
deprioritised - `aria-labels` and semantic HTML will be used as I go along with developing MVP, but tabbing and any other accessibility requirements will be delayed until post-release

**Comment by henrylin03** [comment]
currently relearning accessibility through odin project: https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-introduction-to-web-accessibility

**Comment by henrylin03** [comment]
closing ticket for now - #53 will be tracked separately. further accessibility improvements to have their own tickets.
